hw_1.1/hw_1.1/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+
+class Program
+{
+    static void swap(int[] array, int firstIndex, int secondIndex)
+    {
+        int temp = array[firstIndex];
+        array[firstIndex] = array[secondIndex];
+        array[secondIndex] = temp;
+    }
+
+    static void BubbleSort(int[] array)
+    {
+        for (int i = 0; i < array.Length - 1; ++i)
+        {
+            for (int j = 0; j < array.Length - i - 1; ++j)
+            {
+                if (array[j] > array[j + 1])
+                {
+                    swap(array, j, j + 1);
+                }
+            }
+        }
+    }
+    static void Main(string[] args)
+    {
+        int length = -1;
+        bool isNumber = false;
+
+        while (length <= 0 || !isNumber)
+        {
+            Console.Write("Enter length of the array:");
+            string number = Console.ReadLine();
+            isNumber = int.TryParse(number, out length);
+            if (!isNumber)
+            {
+                Console.WriteLine("Wrong input. Please, enter positive number");
+            }
+        }
+
+        int[] array = new int[length];
```

Comment:
```suggestion
        var array = new int[length];
```
Тип очевиден из правой части